app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -108,13 +116,13 @@ def two_factor_authentication_method
   # Method will be renamed in the next refactor.
   # You can pass in any "type" with a corresponding I18n key in
   # two_factor_authentication.invalid_#{type}
-  def handle_invalid_otp(type: 'otp')
+  def handle_invalid_otp(type:, context: nil)
```

Comment:
I am weirdly delighted that we're able to get rid of the hard-coded 'otp' default here. 👏